### ISSUE TITLE
Clarification/Stressing of do not close the tab.

### ISSUE THREAD

**Issue by Arch881010** [hackermondev/discord-active-developer#3]
I am not that great at coding but I suggest in both 'README.md' and for the done replacement in 'app.js' should be more stressed out.
`DONE | Application/Bot is up and running. DO NOT CLOSE THIS TAB UNLESS YOU ARE FINISHED USING THE BOT, IT WILL PUT THE BOT OFFLINE.` would be better to let them know to not close it out. Maybe rewording would be used.

**Comment by kunaltak606** [comment]
Agreed with odds <3

**Comment by hackermondev** [comment]
done
